notebooks/analysis/requirements.in

```diff
@@ -1,2 +1,2 @@
-weco-datascience
-elasticsearch
+weco-datascience==0.1.9
+elasticsearch==7.14
```

Comment:
Does this need to compiled into a full requirements.txt that pins everything?

Reply:
It does, but that job is currently done by [the dockerfile used by the notebook containers](https://github.com/wellcomecollection/data-science/blob/main/notebooks/Dockerfile#L5-L6) when the container is built